openvdb_houdini/GEO_PrimVDB.h

```diff
@@ -712,7 +736,7 @@ callTypedGrid(GEO_PrimVDB& prim, OpType& op)
 {
     prim.makeGridUnique();
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
@e4lam mentioned that this conditional logic might not be needed any more now that our minimum is Visual Studio 17. I haven't investigated this yet though, do you have any further insights on this?

Reply:
Good point, I'll pass it on to him.